src/Component/Detail/DetailTodoList.js

```diff
@@ -0,0 +1,217 @@
+import CONSTANT from '../../Constants/Constans.js';
+import Component from '../../core/Component.js';
+import { delegate, on } from '../../util/helpers.js';
+import DetailButton from './DetailButton.js';
+
+export default class DetailTodoList extends Component {
+  constructor($element, props) {
+    super($element, props);
+
+    this.members = this.props.members;
+    this.addTeamTodoItem = this.props.addTeamTodoItem;
+    this.changeTeamTodoItemPriority = this.props.changeTeamTodoItemPriority;
+    this.toggleTeamTodoItem = this.props.toggleTeamTodoItem;
+    this.deleteTeamTodoItem = this.props.deleteTeamTodoItem;
+    this.editTeamTodoItemContents = this.props.editTeamTodoItemContents;
+    this.deleteTeamTodoItemAll = this.props.deleteTeamTodoItemAll;
+    this.changeFilter = this.props.changeFilter;
+    this.addTeamMember = this.props.addTeamMember;
+
+    this.templates = new Templates();
+    this.render();
+  }
+
+  handleKeyUp({ key, target }) {
+    const contents = target.value;
+
+    if (key === 'Enter' && contents.length > 0) {
```

Comment:
```suggestion
    if (key === CONSTANT.ENTER && contents.length) {
```
`length`가 0이 아닌 조건이라면 이런식으로도 사용 할 수 있을 것 같아요.